setup.py

```diff
@@ -26,6 +26,37 @@ def read(filename):
         return f.read()
 
 
+def read_reqs(filename):
+    return read(filename).strip().splitlines()
+
+
+install_requires = read_reqs('etc/requirements.txt')
```

Comment:
No concerns about the behavior, but do you think it would be clearer for people if `install_requires` and `extras_require` were both functions or both lists of requirements?